index.bs

```diff
@@ -547,6 +557,7 @@ the {{CredentialCreationOptions}} dictionary as follows:
 <pre class="idl">
     partial dictionary CredentialCreationOptions {
         MakePublicKeyCredentialOptions      publicKey;
+        AbortSignal                         signal;
```

Comment:
hm, [webappsec-credential-management/#dictdef-credentialcreationoptions](https://w3c.github.io/webappsec-credential-management/#dictdef-credentialcreationoptions)  already has a s`signal` member of type `AbortSignal`

Reply:
I added this to deal with a bikeshed linking error. I can take it out.